source/sessions/tests/unified/snapshot-sessions.json

```diff
@@ -0,0 +1,706 @@
+{
+  "description": "snapshot-sessions",
+  "schemaVersion": "1.0",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.9.0",
```

Comment:
But no one needs to, so make it 5.0 to avoid confusion for people that don't understand our wacky versioning scheme (which is changing after 5.0 anyway)

Reply:
Done.